backend/src/main/java/com/praksa/auction/controller/CategoryController.java

```diff
@@ -0,0 +1,28 @@
+package com.praksa.auction.controller;
+
+import com.praksa.auction.model.Category;
+import com.praksa.auction.service.CategoryService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+
+@RestController
+@RequestMapping("/api")
```

Comment:
Is using the same value : **"/api"** above each controller really necessary ? Could we use "/categories" here in RequestMapping above controller, and in the method **getCategories** use only GetMapping() ? If we can, change it for other controllers too.